react-file-uploads/client/app.jsx

```diff
@@ -16,6 +16,22 @@ export default class App extends React.Component {
   }
 
   handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData();
+    formData.append(this.state.caption, this.fileInputRef.current.files[0]);
```

Comment:
The formData object needs two fields appended to it, one named "caption", and one named "image". Currently you are appending only one field where the name _is_ the caption value and the value is the image file.

Reply:
![Screen Shot 2022-09-08 at 5 22 33 PM](https://user-images.githubusercontent.com/106217355/189247697-e41ce7d8-38a8-4968-84e8-97dce08e21b6.png)

![Screen Shot 2022-09-08 at 5 22 51 PM](https://user-images.githubusercontent.com/106217355/189247716-bdace41c-6c29-4df0-9881-07c4ee1bc489.png)

<img width="780" alt="Screen Shot 2022-09-08 at 5 26 27 PM" src="https://user-images.githubusercontent.com/106217355/189248045-a4c2f503-bd53-4bca-9174-8cdc61cc7315.png">